frontend/ui/widget/inputdialog.lua

```diff
@@ -336,6 +336,16 @@ function InputDialog:init()
             self._top_line_num, self._charpos = top_line_num, charpos
         end
     end
+    self.enter_callback = self.enter_callback or function()
+            for _,btn_row in ipairs(self.buttons) do
+                for _,btn in ipairs(btn_row) do
+                    if btn.is_enter_default then
```

Comment:
Too much indentation on the first "for".
Also, it was already bad in the original code you moved here, but please add the missing space after the comma: `_, btn`

Reply:
Fixed.